client/src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from "redux";
+import cityReducer from "./cityReducer";
+import helloReducer from "./helloReducer";
+import itineraryReducer from "./itineraryReducer";
+import activityReducer from "./activityReducer";
+import commentReducer from "./commentReducer";
+
+const rootReducer = combineReducers({
```

Comment:
This could be rewritten as below but there would be some variable changes across the app.
```
const rootReducer = combineReducers({
  cityReducer,
  helloReducer,
  itineraryReducer,
  activityReducer,
  commentReducer
});
```